tools/bliss-to-ogg-zip.py

```diff
@@ -209,9 +342,16 @@ def main():
       if os.path.exists(dest_filename):
         print("already exists, delete: %s" % os.path.basename(dest_filename))
         os.remove(dest_filename)
-      cmd = ["ffmpeg", "-i", rec_filename, "-ss", str(seq.start_time), "-t", str(duration), dest_filename]
+      cmd = ["ffmpeg", "-i", source_filename, "-ss", str(start_time), dest_filename, "-loglevel", args.ffmpeg_loglevel]
+      if not duration == Decimal('Infinity'):
+        cmd[5:5] = ["-t", str(duration)]
```

Comment:
Don't do that. This is error prone (and ugly) for many reasons:
* You should never write `not x == y` but rather `x != y` (btw, PyCharm tells you that, or not?).
* The check is error prone and usual, as you usually would have set something to `None` to declare that it is unset/unused.
* Never hardcode fixed numbers (5 here).

Rather, the code better should be written that way:
```
cmd = ["ffmpeg", "-i", source_filename]
if start_time:
  cmd += ["-ss", str(start_time)]
if duration is not None:
  cmd += ["-t", str(duration)]
cmd += [dest_filename, "-loglevel", args.ffmpeg_loglevel]
```